lib/fedex_rest_api/tracking_object.rb

```diff
@@ -0,0 +1,25 @@
+require "fedex_rest_api/base"
+
+class FedexRestApi::TrackingObject
+  attr_reader :access_token, :include_detailed_scans, :trackable_numbers
+
+  def initialize(tracking_params)
+    @access_token = tracking_params[:access_token] || ""
+    @include_detailed_scans = tracking_params[:include_detailed_scans] || false
+    @trackable_numbers = tracking_params[:trackable_numbers] || []
+  end
+
+  def tracking_numbers
+    tracking_objects = []
```

Comment:
I think we can get the same result using `map`:
```ruby
trackable_numbers.map do |tracking_number|
  {
    "trackingNumberInfo": {
      "trackingNumber": tracking_number
    }
  }
end
```

Reply:
Dig it cause it's way cleaner. Thinking about this being used for public consumption and wanted to be overly clear about the return needing to be an _array of hashes_. I guess specs and a Readme could highlight those nuances just as clearly. Will update!